src/test/java/org/llorllale/cactoos/matchers/HasValuesMatchingTest.java

```diff
@@ -47,10 +47,10 @@
      */
     @Test
     public void matches() {
-        new Assertion<>(
-            "The matcher give the positive result for the valid arguments",
-            () -> new ListOf<>(1, 2, 3),
-            new HasValuesMatching<>(value -> value > 1 || value == 3)
+        new Assertion2<>(
+            "matches iterable with any elements that satisfy predicate",
+            new HasValuesMatching<>(value -> value > 1 || value == 3),
```

Comment:
`value > 1 || value == 3` ?
isn't 3 > 1 always?

Reply:
@ilyakharlamov that was not the point, but I simplified it to `value -> value == 3` anyway